packages/vx-demo/src/components/tiles/Legends.tsx

```diff
@@ -66,191 +65,188 @@ const shapeScale = scaleOrdinal<string, React.FC | React.ReactNode>({
+                  <circle fill={label.value} r={size / 2} cx={size / 2} cy={size / 2} />
+                </svg>
+                <LegendLabel align="left" margin="0 4px">
+                  {label.text}
+                </LegendLabel>
+              </LegendItem>
+            );
+          })
+        }
+      </LegendLinear>
+    </LegendDemo>
+    <LegendDemo title="Threshold">
+      <LegendThreshold scale={thresholdScale}>
+        {labels =>
+          labels.reverse().map((label, i) => {
+            const size = 15;
+            return (
+              <LegendItem
+                key={`legend-quantile-${i}`}
+                margin="1px 0"
+                onClick={() => {
+                  if (events) alert(`clicked: ${JSON.stringify(label)}`);
+                }}
+              >
+                <svg width={size} height={size}>
+                  <rect fill={label.value} width={size} height={size} />
+                </svg>
+                <LegendLabel align="left" margin="2px 0 0 10px">
+                  {label.text}
+                </LegendLabel>
+              </LegendItem>
+            );
+          })
+        }
+      </LegendThreshold>
+    </LegendDemo>
+    <LegendDemo title="Ordinal">
+      <LegendOrdinal scale={ordinalColorScale} labelFormat={label => `${label.toUpperCase()}`}>
+        {labels => (
+          <div style={{ display: 'flex', flexDirection: 'row' }}>
+            {labels.map((label, i) => {
               const size = 15;
               return (
                 <LegendItem
                   key={`legend-quantile-${i}`}
+                  margin="0 5px"
                   onClick={() => {
                     if (events) alert(`clicked: ${JSON.stringify(label)}`);
                   }}
                 >
-                  <svg width={size} height={size} style={{ margin: '2px 0' }}>
-                    <circle fill={label.value} r={size / 2} cx={size / 2} cy={size / 2} />
+                  <svg width={size} height={size}>
+                    <rect fill={label.value} width={size} height={size} />
                   </svg>
-                  <LegendLabel align="left" margin="0 4px">
+                  <LegendLabel align="left" margin="0 0 0 4px">
                     {label.text}
                   </LegendLabel>
                 </LegendItem>
               );
-            })
-          }
-        </LegendLinear>
-      </LegendDemo>
-      <LegendDemo title="Threshold">
-        <LegendThreshold scale={thresholdScale}>
-          {labels =>
-            labels.reverse().map((label, i) => {
+            })}
+          </div>
+        )}
+      </LegendOrdinal>
+    </LegendDemo>
+    <LegendDemo title="Custom Legend">
+      <Legend<string, React.FC | React.ReactNode, typeof shapeScale> scale={shapeScale}>
+        {labels => (
+          <div style={{ display: 'flex', flexDirection: 'row' }}>
+            {labels.map((label, i) => {
               const size = 15;
```

Comment:
can pull the constant out + ALL_CAPS.

Reply:
harry used this pattern a few places, I agree they could be pulled out 👍